### DISCUSSION TITLE
Welcome to udemy-downloader-gui Discussions!

### DISCUSSION THREAD

**Discussion by heliomarpm** [heliomarpm/udemy-downloader-gui#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cgolib** [comment]
Hola, gracias por exelente programa.

**Comment by ymadrigalv** [comment]
Hola tengo la ultima version y me esta dando este error, alguna sugerencia _Contains DRM Protection and cannot de downloaded_
Udemy business

**Reply by heliomarpm** [reply]
Hola, este mensaje se ha agregado para informarle que el curso está protegido contra la piratería y no es posible descargarlo.

**Comment by Azr313** [comment]
Hello Friend,

DRM Protected videos Not Downloading and getting below error. 
![image_2022-07-23_104642464](https://user-images.githubusercontent.com/66358687/180591569-09875b54-88bc-4157-8f15-b2b23e0f6564.png)

**Reply by heliomarpm** [reply]
Hello, this message has been added to inform you that the course is protected against piracy, and it is not possible to download

**Comment by cgolib** [comment]
Usa la versión anterior primero. Intenta descargar,

**Reply by espaol** [reply]
It doesn't work anymore, I have tried a lot to login with gmail but it doesn't anymore. Anyone faced something like that?

**Reply by heliomarpm** [reply]
login with google has really been down for a while, I'll look at this problem again soon to try to solve it.
Login with username and password continues to work perfectly, another alternative is to connect with your token.
